src/apps/vpn/notificationhandler.cpp

```diff
@@ -127,28 +127,23 @@ void NotificationHandler::showNotification() {
           // https://github.com/mozilla-mobile/mozilla-vpn-client/issues/1719
           return;
         }
-
-        //% "VPN Switched Servers"
-        title = qtTrId("vpn.systray.statusSwitch.title");
-        //% "Switched from %1, %2 to %3, %4"
-        //: Shown as message body in a notification. %1 and %3 are countries, %2
-        //: and %4 are cities.
-        message = qtTrId("vpn.systray.statusSwtich.message")
-                      .arg(localizedPreviousExitCountryName,
-                           localizedPreviousExitCityName, localizedCountryName,
-                           localizedCityName);
+        // "VPN Switched Servers"
+        title = L18nStrings::instance()->t(
+            L18nStrings::NotificationsVPNSwitchedServersTitle);
+        message = L18nStrings::instance()
+                      ->t(L18nStrings::NotificationsVPNSwitchedServersMessage)
+                      .arg(localizedPreviousExitCityName, localizedCityName);
       } else {
         if (!SettingsHolder::instance()->connectionChangeNotification()) {
           // Notifications for ConnectionChange are disabled
           return;
         }
-        //% "VPN Connected"
-        title = qtTrId("vpn.systray.statusConnected.title");
```

Comment:
Never mind, this PR is touching the 2 strings that are failing. Still, I don't understand why we haven't seen this type of failure before.

I see `lxml` was [removed recently](https://github.com/mozilla-mobile/mozilla-vpn-client/commit/a977cd0738092d6413ef6146387befd78630b082), but that doesn't seem to be causing the issue.

Technically, I suppose you could try to look at `oldsource` is `source` is None
https://github.com/mozilla-mobile/mozilla-vpn-client/blob/main/.github/l10n/check_l10n_issues.py#L36

Reply:
`vpn.systray.statusConnected.title` is actually used in `src/apps/vpn/platforms/android/androidnotificationhandler.cpp`. Can you please update that file too?